packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -387,41 +387,35 @@ const Dashboard: FC = () => {
         useState<string>();
 
     useEffect(() => {
-        const checkReload = (event: BeforeUnloadEvent) => {
```

Comment:
IIRC this useEffect was supposed to block page refresh (F5)  I don't think this is happening now

Reply:
Yea, I think we need `beforeunload` for this to work correctly with reload. Back button seems to be handled in in Chrome at least.

Reply:
Made the change which shows a warning when user tries to reload